python/pyspark/sql/tests.py

```diff
@@ -3036,6 +3052,9 @@ def test_toPandas_arrow_toggle(self):
         pdf = df.toPandas()
         self.spark.conf.set("spark.sql.execution.arrow.enable", "true")
         pdf_arrow = df.toPandas()
+        # need to remove timezone for comparison
+        pdf_arrow["7_timestamp_t"] = \
+            pdf_arrow["7_timestamp_t"].apply(lambda ts: ts.tz_localize(None))
```

Comment:
Yes, this test is verify behavior is the same with/without Arrow - that is why I made it - and I think we all agree that is the goal.  I'm trying to understand if you are suggesting that this new conf, only if enabled, will add the session local timezone to the Pandas DataFrame made in 
 `toPandas()` with and without Arrow enabled?